src/Collection/CommitteeMembershipCollection.php

```diff
@@ -66,7 +66,7 @@ public function getCommitteeHostMemberships(int $withSupervisors = self::INCLUDE
     public function getCommitteeFollowerMemberships(): self
     {
         return $this->filter(function (CommitteeMembership $membership) {
-            return $membership->isFollower();
+            return $membership->isFollower() && !$membership->getCommittee()->isPending();
```

Comment:
What do you think to rename the `getCommitteeFollowerMemberships`method to `getCommitteeFollowerMembershipsInProcess` ?

Reply:
@thomasbisignani the actual name seems good. It's the opposite, we check that the committee status is not pending.